docs/usage/20-provisioning-ingresses.adoc

```diff
@@ -51,69 +51,102 @@ In case an invalid `Service` resource is specified as a backend for a given `Ing
 Whenever `dklb` gets to be used as a backend, a Kubernetes event will be emitted and associated with the `Ingress` resource being processed.
 This event contains useful information about the reason why `dklb` is being used instead of the intended backend, and may be used for diagnosing problems.
 
+=== Customizing the target EdgeLB pool
+
+As mentioned before, `dklb` uses sane defaults when provisioning EdgeLB pools for `Ingress` resources annotated for provisioning with EdgeLB.
+It is, however, possible to customize the target EdgeLB pool for a given `Ingress` resource by using the `kubernetes.dcos.io/dklb-config` annotation.
+This annotation accepts a YAML object (henceforth called the _configuration object_) with the following structure:
+
+[source,yaml]
+----
+kubernetes.dcos.io/dklb-config: |  # NOTE: The "|" character is mandatory.
+  name: dklb-pool-0
+  role: "*"
+  network: "dcos"
```

Comment:
Why are _role_ and _network_ double-quoted when _name_ is not?

Reply:
_Role_ must be double-quoted in order to indicate to the YAML parser that what follows is a string. _Network_ doesn't need to be, because it starts with an alphabetic character. I'll drop the double-quotes there.